openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -688,6 +747,17 @@ def retire_sapsf_data_transmission(user):
                 )
                 audits.update(sapsf_user_id='')
 
+    @staticmethod
+    def retire_degreed_data_transmission(user):
+        for ent_user in EnterpriseCustomerUser.objects.filter(user_id=user.id):
+            for enrollment in EnterpriseCourseEnrollment.objects.filter(
+                enterprise_customer_user=ent_user
+            ):
+                audits = DegreedLearnerDataTransmissionAudit.objects.filter(
+                    enterprise_course_enrollment_id=enrollment.id
+                )
+                audits.update(degreed_user_email='')
```

Comment:
@iloveagent57 wrote a ticket about moving these types of enterprise deletions to the edx-enterprise Django app: https://openedx.atlassian.net/browse/EDUCATOR-2813
Could you ensure that moving this code is represented there as well? The accompanying PR was:
https://github.com/edx/edx-platform/pull/18060

Reply:
Either someone recently edited that ticket or it's already noted there: 

> Please also make sure the model DegreedLearnerDataTransmissionAudit is also included in the retirement code.